### DISCUSSION TITLE
Deleting published scenes from Spoke.

### DISCUSSION THREAD

**Discussion by JT-os** [mozilla/hubs#3167] [Spoke Questions] [answered]
If I publish something with Spoke and don't check promote, then I decide I want to delete it, how do I go about doing that?
If I back up to projects will it be there and I can delete it from there?
Thanks in advance.

**Comment by ghost** [comment, marked as answer]
Right-clicking from the Projects page and selecting 'delete' worked for me.

**Reply by JT-os** [reply]
Thanks.  I was trying to delete from inside of Spoke's editor was the issue.  Once I got back out to the projects page is where I saw the option.  Thanks again : )
